libvcx/src/api_lib/api_c/wallet.rs

```diff
@@ -1345,7 +1345,7 @@ pub mod tests {
         close_main_wallet().await.unwrap();
     }
 
-    #[tokio::test]
+    #[tokio::test(flavor = "multi_thread")]
```

Comment:
Just curious, why is this?

Reply:
This particular test requires a multi-thread executor. The last `cb.receive(..).unwrap()` fails with timeout on a single thread executor.